app/src/screens/MapScreen.js

```diff
@@ -111,25 +131,37 @@ function MapScreen(): React.Node {
     Object.keys(unfilteredLayers).forEach(key => {
       newLayers[key] = {...unfilteredLayers[key]};
       newLayers[key].data = newLayers[key].data.filter(point => {
-        if (filterMinRssi != '' && point.rssi < filterMinRssi) {
+        if (filterMinRssi !== '' && point.rssi < filterMinRssi) {
```

Comment:
Does flow not complain about the types here?  point.rssi is a number and filterMinRssi is a string.

Reply:
Part of the original filter code, but yea it just works as JS converts string to number